### ISSUE TITLE
[Accessibility] There is no way to activate and move widget using move button on the widget

### ISSUE THREAD

**Issue by zoarif** [ckeditor/ckeditor4#4814]
There should be a way to move widget using the keyboard. I tried but enter or space does not activate the move. 
![image](https://user-images.githubusercontent.com/18426614/128782593-f3388b5a-b86e-4f09-9230-022b1c254e36.png)
I am using it in my application but the button is inaccessible. I can't make changes to this button as it is CK internal code.

**Comment by Comandeer** [comment]
According to [WCAG's Success Criterion 2.1.1. Keyboard](https://w3c.github.io/wcag/guidelines/#keyboard), actions on the page should be available from keyboard, however, as it's described in more detail in both [technique to satisfy this criterion](https://www.w3.org/WAI/WCAG21/Techniques/general/G202.html) and [its section in "Understanding WCAG" note](https://w3c.github.io/wcag/understanding/keyboard.html), it does not mean that the same method of doing an action must be used for both pointer and keyboard.

"Understanding WCAG" guide provides even the example of how d&d feature can be implemented for keyboard:

> **Example 2: A drag and Drop Feature.**
> 
> An application that uses drag and drop also supports "cut" and "paste" or form controls to move objects.

And that's exactly how CKEditor 4 implements support for moving widgets: via cut and paste.
